record-and-playback/core/scripts/utils/captions.rb

```
@@ -54,7 +54,7 @@
 def create_api_captions_file(captions_meeting_dir)
   BigBlueButton.logger.info("Generating closed captions for API")
 
-  captions = JSON.load(File.new("#{captions_meeting_dir}/captions_playback.json"))
+  captions = JSON.load(File.read("#{captions_meeting_dir}/captions_playback.json"))
```

RuboCop (1.86.2)

[convention] Prefer `JSON.parse` over `JSON.load`.

(Security/JSONLoad)